jobs/payment-jobs/invoke_jobs.py

```diff
@@ -58,8 +58,9 @@ def create_app(
     app.logger.info("<<<< Starting Payment Jobs >>>>")
     queue.init_app(app)
     db.init_app(app)
-    if init_oracle:
-        oracle_db.init_app(app)
+    if init_data_warehouse:
```

Comment:
when wouldn't you init for datawarehouse?

Reply:
some jobs no need init for datawarehouse.
```
    jobs_with_data_warehouse_connections = ["BCOL_REFUND_CONFIRMATION"]
    application = create_app(job_name=job_name, init_data_warehouse=job_name in jobs_with_data_warehouse_connections)
```

Reply:
I think it's fine to just run it for any job